ember_debug/adapters/basic.js

```diff
@@ -113,7 +113,7 @@ export default EmberObject.extend({
   */
   connect() {
     return new Promise((resolve, reject) => {
-      $(() => {
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
where to? if other repositories depend on it, it might just be as interesting to use a npm package for this, don't you think?